tests/installation/add_update_test_repo.pm

```diff
@@ -33,7 +33,12 @@ sub run() {
         send_key $cmd{next};
         assert_screen 'addon-products';
         # if more repos to come, add more
-        send_key 'alt-a' if @repos;
+        if (@repos) {
+            send_key 'alt-a';
+            until (check_screen('addon-menu-active', 5)) {
```

Comment:
I'm with @kravciak here, as job will fail in 2 hours instead of 2 minutes let's say. I would even use assert screen here, so in case of new interface we see immediately that needles requires adjustment.